docs/questions/visualizations/visualizing-results.md

```diff
@@ -9,6 +9,26 @@ redirect_from:
 
 While tables are useful for looking up information or finding specific numbers, it's usually easier to see trends and make sense of data using charts.
 
+## Visualize query results
+
+### Visualizing questions in the query builder
+
+To visualize results of a question built in the [query builder](../query-builder/editor.md), click on the **Visualize** button under the last query builder step. Metabase will select a chart type most appropriate for your data, but you can [change the visualization type](#change-visualization-type).
```

Comment:
```suggestion
To visualize results of a question built in the [query builder](../query-builder/editor.md), click on the **Visualize** button under the last query builder step. Metabase will select a chart type most appropriate for your data, but you can [change the visualization type](#change-visualization-type). You can also toggle between the visualization and the table of results.
```

Just thought we should mention the results table, as that's a given for native queries.